### PR TITLE
tests(next): added tests for path-utils

### PR DESCRIPTION
**Pull request by anikethsaha**
**What kind of change does this PR introduce?**

Tests

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
NA

**Summary**
Added tests for `path-utils` 

**Does this PR introduce a breaking change?**
No

**Other information**
NA

### CODE REVIEW

**Comment by anikethsaha** [comment]
cc @evenstensberg

File: packages/utils/__tests__/path-utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { join } from "path";
+import { isLocalPath, findProjectRoot } from "../path-utils";
+
+describe("path-utils tests", () => {
+  it("isLocalPath Testing : __dirname", () => {
+    const answer = isLocalPath(join(__dirname, "../../"));
```
Comment (ematipico) [inline packages/utils/__tests__/path-utils.test.ts]: You should test this against different formats:
- ./path
- path/path
- ..\path (win)

And also you should test the error cases

**Comment by anikethsaha** [comment]
I have submitted a PR https://github.com/webpack/webpack-cli/pull/999 regarding the `isLocalPath` method which seems to be buggy. tests in this commit https://github.com/webpack/webpack-cli/pull/984/commits/b9c4260c40d50a3ae68a4a9a4b7f02554a3d162d will be passed if https://github.com/webpack/webpack-cli/pull/999 is considered cause currently for non-valid local path the `isLocalPath` will return `true`

@ematipico @evenstensberg PTAL

**Comment by evenstensberg** [comment]
We cannot merge anything before the tests are passing, so we need to fix those first

**Pull request closed without merging**